tests/test_meshtastic_utils.py

```
@@ -5146,6 +5159,235 @@ def test_connect_meshtastic_ble_interface_creation_timeout(
+
+        config = {
+            "meshtastic": {
+                "connection_type": CONNECTION_TYPE_BLE,
+                "ble_address": TEST_BLE_MAC,
+                "retries": 1,
+            },
+            "matrix_rooms": [],
+        }
+
+        class _BleInterfaceWithAutoReconnect:
+            def __init__(
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)